README.md

```diff
@@ -1,37 +1,48 @@
-# Nimiq Style
+# Nimiq Style Framework
 
-The Nimiq CSS framework
+The Nimiq CSS framework, defining UI look and feel: colors, fonts, sizes.
 
-## Documentation
+**Please refer to the [demo page](demo.html)**
+for examples and in-depth explanations of available UI components, how they look like, and how to use them.
 
-[Go to documentation with examples.](https://naughty-raman-53eb55.netlify.com)
+## Add Nimiq Style to your project
 
-## Include via NPM
+The style framework consists of two parts, the CSS definitions and
+the fonts, available via Google Fonts.
 
+### Add the CSS
```

Comment:
I don't think going down to H4 in a Readme is a good choice and I'd say it's best maximally use H3. Thus I think this headline is actually unecessary, because this repo only has CSS files to install. There is no other sibling heading for JS or HTML or similar.